WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -1,22 +1,24 @@
 using NBitcoin;
 using ReactiveUI;
-using Splat;
 using System;
 using System.Reactive;
+using System.Reactive.Concurrency;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
-using WalletWasabi.Blockchain.TransactionBroadcasting;
+using System.Threading.Tasks;
 using WalletWasabi.Fluent.ViewModels.Wallets.HardwareWallet;
+using WalletWasabi.Fluent.ViewModels.Wallets.History;
 using WalletWasabi.Fluent.ViewModels.Wallets.WatchOnlyWallet;
 using WalletWasabi.Gui;
 using WalletWasabi.Logging;
+using WalletWasabi.Stores;
 using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets
 {
 	public partial class WalletViewModel : WalletViewModelBase
 	{
-		protected WalletViewModel(UiConfig uiConfig, Wallet wallet) : base(wallet)
+		protected WalletViewModel(UiConfig uiConfig, Wallet wallet, BitcoinStore bitcoinStore) : base(wallet)
```

Comment:
dont pass bitcoinstore, we are already passing `Wallet` and that has a `BitcoinStore` property,

Reply:
done